lib/plugins/aws/provider.js

```diff
@@ -1510,6 +1510,10 @@ class AwsProvider {
     return `${provider.deploymentPrefix}`;
   }
 
+  getCustomDeploymentRole() {
+    return this.serverless.service.provider.cfnRole;
```

Comment:
@medikoo due to the nature of tests, initialising say `provider.customDeploymentRole` property in constructor will be a lot more invasive - lots of tests will need to be changed because of that. Let me know what you think.

Reply:
Can you list which test files will crash? We should schedule all of them for refactor to `runServerless` variant, having that such change won't influence unexpected test fails.